packages/libs/client/src/tests/pact.test.ts

```diff
@@ -197,4 +197,98 @@ describe('Pact proxy', () => {
 
     expect(() => builder.poll('fake-api-host.local.co')).toThrow();
   });
+
+  describe('unfinishes the transaction', () => {
```

Comment:
`describe` contains the subject, `it` contains the action that is executed and the result that it should yield

I this case `describe('Pact proxy')` `it('unfinishes the transaction when setMeta is called')`

gives this sentence: `Pact proxy unfinishes the transaction when setMeta is called`